prime-dotnet-webapi/Models/EnrolleeLinkedEnrolment.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Newtonsoft.Json;
+
+namespace Prime.Models
+{
+    [Table("EnrolleeLinkedEnrolment")]
+    public class EnrolleeLinkedEnrolment : BaseAuditable
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [JsonIgnore]
+        public Enrollee Enrollee { get; set; }
+
+        public int EnrolleeId { get; set; }
+
+        [JsonIgnore]
+        public Enrollee PaperEnrollee { get; set; }
+        public int? PaperEnrolleeId { get; set; }
+        public string UserProvidedGpid { get; set; }
+        public DateTime EnrolmentLinkDate { get; set; }
+        public bool IsConfirmed { get; set; }
```

Comment:
"Confirmed" is enough to indicate that the property is a bool, I feel. Compare to `Submission`s, which have an analogous property `Confirmed`.

Reply:
Renamed